### PR TITLE
Sprint20 gds compliance

### PR DESCRIPTION
**Pull request by DH02**
HTML/Views updated to GDS/CDN.

### CODE REVIEW

File: src/SFA.DAS.EmployerAccounts.Web/Views/EmployerAccountPaye/Details.cshtml [hunk 1]
```diff
@@ -11,83 +11,78 @@
...
-                    <th class="numeric" scope="col"><span class="nowrap">English %</span></th>
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+        <table class="govuk-table">
+            <caption class="govuk-table__caption govuk-heading-m">Scheme history</caption>
+            <thead class="govuk-table__head">
+                <tr class="govuk-table__row">
+                    <th class="govuk-table__header" scope="col">Date</th>
+                    <th class="govuk-table__header" scope="col"><span class="das-no-wrap">English %</span></th>
```
Comment (jamesmking) [inline src/SFA.DAS.EmployerAccounts.Web/Views/EmployerAccountPaye/Details.cshtml]: Should this have an extra class of 'govuk-table__header--numeric' to right align it?

File: src/SFA.DAS.EmployerAccounts.Web/Views/EmployerAccountPaye/Details.cshtml [hunk 2]
```diff
@@ -11,83 +11,78 @@
...
-                                <span>@fraction.DateCalculated.ToString("dd MMM yy")</span>
+                        <tr class="govuk-table__row">
+                            <td class="govuk-table__cell">
+                                <span class="das-no-wrap">@fraction.DateCalculated.ToString("dd MMM yy")</span>
                             </td>
-                            <td class="numeric">
-                                @if (fraction.Amount == 1)
-                                {
-                                    <span>100%</span>
+                            <td class="govuk-table__cell">
```
Comment (jamesmking) [inline src/SFA.DAS.EmployerAccounts.Web/Views/EmployerAccountPaye/Details.cshtml]: And the <td>s have an extra class of 'govuk-table__cell--numeric' to right align the numbers

File: src/SFA.DAS.EmployerAccounts.Web/Views/Home/Index.cshtml [hunk 3]
```diff
@@ -8,72 +8,76 @@
...
 @if (!string.IsNullOrWhiteSpace(Model.Data.ErrorMessage))
 {
-    <div class="grid-row">
-        <div class="column-full">
-            <div class="panel panel-error">
-                @Model.Data.ErrorMessage
-            </div>
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+        <div class="panel panel-error">
```
Comment (jamesmking) [inline src/SFA.DAS.EmployerAccounts.Web/Views/Home/Index.cshtml]: .panel and .panel-error don't exist in the CDN, should these be updated?

File: src/SFA.DAS.EmployerAccounts.Web/Views/Home/Index.cshtml [hunk 4]
```diff
@@ -8,72 +8,76 @@
...
-                        </th>
-                    </tr>
-                </thead>
-                <tbody>
-                    @foreach (var account in Model.Data.Accounts.AccountList)
+        <h1 class="govuk-heading-xl" id="Levy-accounts">Your accounts</h1>
+
+        <p class="govuk-body govuk-!-margin-bottom-6">Select an account or add a new one.</p>
+
+        <table class="govuk-table">
```
Comment (jamesmking) [inline src/SFA.DAS.EmployerAccounts.Web/Views/Home/Index.cshtml]: This table needs an extra class of 'das-table--responsive' to make it responsive, so its the same as the current screen

File: src/SFA.DAS.EAS.Web/Views/Shared/_WhatIsEnglishPercentage.cshtml [hunk 5]
```diff
@@ -1,7 +1,11 @@
...
-        <p>Your English percentage is provided by HMRC based on the data in your payroll submissions. It represents the proportion of your payroll that's paid to your employees that live in England.</p>
-        <p>If you think there are any issues with your English percentage, check your payroll data to make sure your employees' address information is correct.</p>
+<details id="inform-english-percentage" class="govuk-details">
+    <summary class="govuk-details__summary">
+        <span class="govuk-details__summary-text">
+            What is English percentage? 
+        </span>
+    </summary>
+    <div class="govuk-details__text">
+        Your English percentage is provided by <abbr>HMRC</abbr> based on the data in your payroll submissions. It represents the proportion of your payroll that's paid to your employees that live in England.<br>
```
Comment (jamesmking) [inline src/SFA.DAS.EAS.Web/Views/Shared/_WhatIsEnglishPercentage.cshtml]: Should this text be wrapped in a `<p>`?

File: src/SFA.DAS.EmployerAccounts.Web/Views/EmployerAccountPaye/Details.cshtml (hunk 1 above)
Comment (DH02) [reply src/SFA.DAS.EmployerAccounts.Web/Views/EmployerAccountPaye/Details.cshtml]: My file doesn't have this class? (.govuk-table__header--numeric)

File: src/SFA.DAS.EAS.Web/Views/Shared/_WhatIsEnglishPercentage.cshtml (hunk 5 above)
Comment (DH02) [reply src/SFA.DAS.EAS.Web/Views/Shared/_WhatIsEnglishPercentage.cshtml]: The GDS pattern shows raw text in the DIV. (Elsewhere, if text is split into two paragraphs I added 2x Ps; along the same principle of text within table cells.

File: src/SFA.DAS.EmployerAccounts.Web/Views/EmployerAccountPaye/Details.cshtml (hunk 1 above)
Comment (DH02) [reply src/SFA.DAS.EmployerAccounts.Web/Views/EmployerAccountPaye/Details.cshtml]: Added.

File: src/SFA.DAS.EmployerAccounts.Web/Views/EmployerAccountPaye/Details.cshtml (hunk 2 above)
Comment (DH02) [reply src/SFA.DAS.EmployerAccounts.Web/Views/EmployerAccountPaye/Details.cshtml]: Added. (Looks like the if/else only had numeric on one, so now added to both TDs.

File: src/SFA.DAS.EmployerAccounts.Web/Views/Home/Index.cshtml (hunk 3 above)
Comment (DH02) [reply src/SFA.DAS.EmployerAccounts.Web/Views/Home/Index.cshtml]: My excuse for this one is that no errors are fired on this page (and it appears - as confirmed by Chris - that this is/was rouge code in the file). This redundant "error message HTML" has now been removed completely.

**Pull request merged**